store/NoOp.go

```diff
@@ -0,0 +1,25 @@
+package store
+
+func NewNoOp() *NoOp {
```

Comment:
Not needed and could use (NoOp) instead of (*NoOp) prefer pointer method receiver when you've data to change.